src/scene/materials/standard-material.js

```diff
@@ -356,21 +349,13 @@ class StandardMaterial extends Material {
     }
 
     reset() {
-        for (let i = 0; i < _propsSerial.length; i++) {
-            const defVal = _propsSerialDefaultVal[i];
-            this[_propsSerial[i]] = defVal ? (defVal.clone ? defVal.clone() : defVal) : defVal;
-        }
-        for (let i = 0; i < _propsInternalNull.length; i++) {
-            this[_propsInternalNull[i]] = null;
+        // set default values
+        for (const [name, prop] of Object.entries(_props)) {
```

Comment:
might need a polyfill for IE https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Reply:
damn, nice catch. I keep expecting babel will sort out this stuff for us. @willeastcott, do you know if it's possible to get babel to handle this for us automatically?

Reply:
using keys.forEach instead